src/commands/buildCommands/configs/serverAppConfig.js

```diff
@@ -104,23 +105,24 @@ function getConfig(localeCode) {
 					: null,
 				whitelist: [
 					/^meetup-web-components/,
-					/^swarm-icons\/dist\/sprite\/sprite\.inc$/,
-				],
+					/^@meetup\//,
```

Comment:
Not sure if it matters, but do you need the trailing slash here? `meetup-web-components` doesn't seem to have one.

Reply:
since `@meetup/` is always a prefix, it's good to enforce the slash to prevent things like `@meetups-evil-twin/example` from matching. For `meetup-web-components`, in theory that could be imported as a whole package without a slash, so it's better to match without the trailing slash (`import MWC from 'meetup-web-components';`)

Reply:
that said, `/^meetup-web-components/` could be `/^meetup-web-components(?:$|\/)/` if we wanted to be sticklers